src/editor/editorinteractive_lua.cc

```diff
@@ -168,9 +169,11 @@ void EditorInteractive::write_lua(FileWrite& fw) const {
 		write("local %s = wl.Game().map", autogen_map_name);
 
 		const uint8_t nrplayers = map.get_nrplayers();
+		std::unique_ptr<std::string[]> pvar_cache(new std::string[nrplayers]);; // so the char*s won't go out of scope at once
```

Comment:
Doesn't this work with a local variable in the loop? Also, double ;;

Reply:
I also thought it would, but ASan has a different opinion…

Reply:
Let's leave it like this then.

Reply:
clang-format already did that automatically :)

Reply:
Yay for awesome tools :)